source/common/http/conn_manager_impl.cc

```diff
@@ -348,6 +349,9 @@ void ConnectionManagerImpl::resetAllStreams() {
     auto& stream = *streams_.front();
     stream.response_encoder_->getStream().removeCallbacks(stream);
     stream.onResetStream(StreamResetReason::ConnectionTermination, absl::string_view());
+    if (response_flag) {
+      stream.stream_info_.setResponseFlag(*response_flag);
```

Comment:
nit: the style almost everywhere on the codebase is to use `has_value()` and `value()`

Reply:
Ah, that is nicer. Thanks